kinetic/distribution.yaml

```diff
@@ -8083,7 +8083,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/astuff/network_interface-release.git
-      version: 2.1.0-0
+      version: 2.1.0-1
```

Comment:
What changed between `-0` and `-1`? Did you want to bump the version and tag a new release instead?

Reply:
@sloretz - I believe this was a mistake. You can close for now. We will be releasing a new version soon but it will be 2.2.0. Thanks.